File: CodeHistory/Monday, June,17,2024-6:18:2 AM.js
function isAnagram(str1, str2) {
    let sortedStr1 = str1.toLowerCase().split('').sort().join('');
    let sortedStr2 = str2.toLowerCase().split('').sort().join('');
    
    return sortedStr1 === sortedStr2;
}

console.log(isAnagram("listen", "silent"));  // true
console.log(isAnagram("hello", "world"));    // false

File: CodeHistory/Monday, June,17,2024-6:18:2 AM.py
def is_anagram(str1, str2):
    sorted_str1 = ''.join(sorted(str1.lower()))
    sorted_str2 = ''.join(sorted(str2.lower()))
    
    return sorted_str1 == sorted_str2

print(is_anagram("listen", "silent"))   # True
print(is_anagram("hello", "world"))     # False

